lib/active_model/serializer/adapter/json.rb

```diff
@@ -7,7 +7,7 @@ class Json < Base
 
         def serializable_hash(options = nil)
           options ||= {}
-          { root => Attributes.new(serializer).serializable_hash(options) }
+          { root => Attributes.new(serializer, instance_options).serializable_hash(options) }
```

Comment:
Not yet, but here, even with virtually no information whatsoever, not passing along the `instance_options` means "no options can ever be set on the Json adapter".